src/HotChocolate/Core/src/Types.Mutations/ResolverArgument.cs

```diff
@@ -0,0 +1,35 @@
+#nullable enable
+
+namespace HotChocolate.Types;
+
+internal sealed class ResolverArgument : IInputFieldInfo
```

Comment:
This represents our resolver argument substitute which we preserve in our mutation middleware.